okta-spring-security-starter/src/main/java/com/okta/spring/oauth/OktaOAuthProperties.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.spring.oauth;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("okta.oauth")
+public class OktaOAuthProperties {
+
+    /**
+     * Custom authorization server issuer URL: i.e. 'https://dev-123456.oktapreview.com/oauth2/ausar5cbq5TRooicu812'.
+     */
+    private String issuer;
+
+    /**
+     * Expected access token audience claim value.
+     */
+    private String audience = "all";
```

Comment:
@mraible @bretterer  before I forget about this. A default audience of 'all' doesn't seem great, this was just what I had configured my Authorization Server with at one point, any ideas on a _better_ default ?